src/tm2/tm-device.h

```diff
@@ -95,9 +95,45 @@ namespace librealsense
         void set_manual_exposure(bool manual);
 
         // Pose interfaces
+
+        /**
+         * Get relocalization map from T265
```

Comment:
```suggestion
         * Get relocalization map that is currently on device, created and updated during most recent tracking session.
         * Can be called before or after stop().
```
I'm not sure these statements are correct, so please fix if they are not. But the idea is to convey 1) that relocalization map is something built during tracking, 2) what period of tracking usage it corresponds to, and 3) whether user needs to stop tracker before exporting. For the second point, it would be great to be precise (but brief) about what period of tracking is reflected in the map, e.g. everything since either most recent import() or start() or whatever. For example, what about cases like import->start->stop->start->stop->export, or start->stop->import->start->stop->export?